data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/handlerinterfaces/HandlerInterface.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.beaconrestapi.handlerinterfaces;
+
+public interface HandlerInterface {
+
+  void run();
```

Comment:
I'm scope creeping a bit here, but I'd be inclined to design this interface so that it can abstract away Javalin as that will make testing a lot easier.  So I'd suggest:

```
public interface BeaconRestApiHandler {
  String getPath();
  String handleRequest(RequestParams params); // Returns JSON response
}

public class RequestParams {
  private final Context javalinContext;
  public RequestParams(Context javalinContext) {
    this.javalinContext = javalinContext;
  }

  public String getQueryParam(String name) {
    return javalinContext.queryParam(name);
  }
}
```

Then in a test, it's easy to mock `RequestParams` and execute the handler without involving Javalin.

Dependencies like `clientStorage` and the P2P network would be passed into the constructors of handlers.  You can then just have this one interface and remove `P2PNetworkHandlerInterface` and `BeaconHandlerInterface`.

`BeaconRestApi` would do the integration with Javalin so would do something like:
```
handlers.forEach(handler -> app.get(handler.getPath(), ctx -> handler.handleRequest(new RequestParams(ctx)));
```